.github/workflows/publish.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish package on NPM
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - run: yarn config set npmAuthToken $YARN_NPM_AUTH_TOKEN
+        env:
+          YARN_NPM_AUTH_TOKEN: ${{ secrets.YARN_NPM_AUTH_TOKEN }}
+      - run: yarn
+      - run: yarn pack
+      - run: yarn publish
```

Comment:
if at some point we want to make beta releases we can with the "prerelease" option in github releases: https://github.com/DataDog/scope-javascript-agent/blob/master/.github/workflows/npmpublish.yml#L25